src/components/Video.js

```diff
@@ -66,6 +67,17 @@ class MuxVideo extends Component {
 
   componentDidMount() {
     this.video = React.createRef()
+
+    const style = document.createElement('style')
+    style.innerHTML = 'button svg { vertical-align: middle; }'
+
+    if (this.playRef.current.shadowRoot) {
+      this.playRef.current.shadowRoot.appendChild(style)
+    }
+    if (this.muteRef.current.shadowRoot) {
+      this.muteRef.current.shadowRoot.appendChild(style.cloneNode(true))
+    }
```

Comment:
yikes... is that the only way to get styles into the `media-container`? If so, we should revisit that on the `media-chrome` side because there should be a better story around applying styles. (Those changes shouldn't be a blocker for this iteration, but something to note for looking at later)

Reply:
Yes, this is indeed a bit cumbersome. I tried a couple of different ways of styling the `media-chrome` player but this was the only way that actually worked.

I am unfortunately no expert on web components, but from what I have read it seems like this is the only way to inject styles inside the shadow-root of components. One way of styling that passes through the shadow root is via CSS variables, which I noticed do exist to some extent inside media-chrome. Unfortunately though, not for the exact CSS we needed to override in this case.

My understanding is that you can opt-out of using a shadow-root altogether and that would allow for styles to pass through, but you would lose the protective encapsulation effect that it also provides.

Reply:
Also, if you or anyone else know of a more clean way, that would be super helpful!